src/components/selectboxes/SelectBoxes.js

```diff
@@ -54,6 +54,7 @@ export class SelectBoxesComponent extends RadioComponent {
   setValue(value, flags) {
     value = value || {};
     flags = this.getFlags.apply(this, arguments);
+    this.data[this.component.key] = value;
```

Comment:
Why doesn't ```updateValue``` take care of this? I am worried that this will break the "changed" triggers from working.